.github/workflows/integration-test.yml

```diff
@@ -55,7 +66,18 @@ jobs:
     with:
       database: ${{ github.event.inputs.database2 }}
       engine: ${{ github.event.inputs.engine2 }}
-      account: ${{ github.event.inputs.account }}
+      account: ${{ vars. FIREBOLT_ACCOUNT_V1 }}
+    secrets:
+      FIREBOLT_CLIENT_ID_STG_NEW_IDN: ${{ secrets.FIREBOLT_CLIENT_ID_STG_NEW_IDN }}
+      FIREBOLT_CLIENT_SECRET_STG_NEW_IDN: ${{ secrets.FIREBOLT_CLIENT_SECRET_STG_NEW_IDN }}
+
+  run-integration-tests-engine2:
+    if: ${{ github.event.inputs.run-database-with-engine-v2 == 'true' }}
+    uses: ./.github/workflows/integration-test-v2.yml
+    with:
+      database: ${{ github.event.inputs.database-with-engine-v2 }}
+      engine: ${{ github.event.inputs.engine_v2_fb_2_0 }}
```

Comment:
Let's take this opportunity to change all `github.event.inputs` to `inputs`. As we saw elsewhere this is a safer option.

Reply:
Done, although making unrelated changes is the classical way to the hell.

Reply:
I'd argue since we're adding these lines and we know them to be working incorrectly in certain cases it's related to this PR. We can either change only these lines but then we'll have two different calls in the same file or change them altogether and have a issue that we don't have to come back to.
This is not a dangerous operation/code that changed so in my opinion it's worth the effort to fix on the spot.